### PR TITLE
DO NOT MERGE YET! - Refactoring + clean up

### PR DESCRIPTION
**Pull request by jleni**
This PR refactors, cleans up, upgrades dependencies and fixes CI before releasing a new NPM package that supports both Cosmos Ledger App v1.5.x and v2.x.x

### CODE REVIEW

File: vue_example/App.vue [hunk 1]
```diff
@@ -1,31 +1,25 @@
...
-      <img
-        alt="Vue logo"
-        src="./assets/logo.png"
-        width="300"
-      >
-    </a>
-    <h1>Cosmos - Ledger integration example</h1>
-    <CosmosLedger msg="Minimal Cosmos+Ledger delegation example" />
+    <img alt="Zondax logo" src="./assets/logo.png" width="300" />
+    <h1>Kusama integration example</h1>
```
Comment (marbar3778) [inline vue_example/App.vue]: lol

File: .circleci/config.yml [hunk 2]
```diff
@@ -9,20 +9,23 @@ jobs:
   build:
     <<: *defaults
     steps:
+      - run: sudo apt-get install libusb-1.0.0
```
Comment (marbar3778) [inline .circleci/config.yml]: note: update circleci config

nothing to be done in this PR, but just a note to myself

File: vue_example/App.vue (hunk 1 above)
Comment (jleni) [reply vue_example/App.vue]: oops :1st_place_medal: :)
That was trying to unify examples/formatting to minimize integration efforts.. !

**Comment by jleni** [comment]
Closing to avoid confusion. I will reopen later today.

**Pull request closed without merging**